From e95e6ae9eeb47fc56156dffbda305c2a61c19883 Mon Sep 17 00:00:00 2001
From: Lijun Fang <fanglijun3@huawei.com>
Date: Thu, 30 Dec 2021 16:26:10 +0800
Subject: [PATCH] svm: Set CONFIG_HISI_SVM as m by default
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: e95e6ae9eeb47fc56156dffbda305c2a61c19883


ascend inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I4JMM0
CVE: NA
-------------------

Set CONFIG_HISI_SVM as m by default

Signed-off-by: Lijun Fang <fanglijun3@huawei.com>
Reviewed-by: Weilong Chen <chenweilong@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index fb5b9650b458..83691c2ad677 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -3300,7 +3300,7 @@ CONFIG_TCG_TIS_ST33ZP24_I2C=y
 CONFIG_TCG_TIS_ST33ZP24_SPI=y
 # CONFIG_XILLYBUS is not set
 CONFIG_PIN_MEMORY_DEV=m
-CONFIG_HISI_SVM=y
+CONFIG_HISI_SVM=m
 # end of Character devices
 
 # CONFIG_RANDOM_TRUST_CPU is not set
-- 
2.26.2

